arcade/experimental/video_player.py

```diff
@@ -9,14 +9,20 @@
 import arcade
 
 
-class VideoPlayer(arcade.Window):
```

Comment:
It would be a good idea to make a separate videoplayer class here that is used by the view itself?

Reply:
Yes on it.